content/Lab2/10_Deploying_Solution/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Deploying the solution in AWS and Cisco Secure Firewall using Terraform"
+weight: 10
+---
+
+In this section, we will use Terraform to deploy the lab in AWS and also configure the deployed Cisco Secure Firewall with the required configuration.
+
+
+[Solution Walkthrough](../10_Deploying_Solution/1_deploying_solution.md)
```

Comment:
please rename the respective md titles so that it matches with this text e.g. 1_network_resources.md title should be AWS Network Resources - Description